README.md

```diff
@@ -1,2 +1,69 @@
-# mens-conference
-The official site for Men's Conference.
+# **Men's Conference**
+The official (not really) site for the 2023 Men's Conference held in Madikwe Game Reserve, South Africa.
+
+# Built With
+- Javascript
+- HTML & CSS
+- HTML & CSS Linters (Lighthouse, Webhint, Stylelint, NodeChecker)
+- Git Bash
+- VS Code
+
+# Live Demo
+[Click here](https://botlhale-setou.github.io/mens-conference/index.html) to view live demo.
+
+# Getting Started
+This is a mock website for a "men only" conference. <br>
+My first capstone project @ Microverse. <br>
+It showcases the author's functional understanding of basic HTML, CSS and JS colaborative intergration.
+
+- **PLEASE COMMIT CHANGES TO NEW BRANCH & SUBMIT PR BEFORE MERGING.** 
+- **PLEASE MAKE NO CHANGES TO "git config"**
+
+## Prerequisites
+- Javascript
+- GitHub Account
+- Git Bash (or other Terminal)
+- VS Code (or other IDE/Text-Editor)
+
+# Setup
+To get a local copy up and running, follow these steps:
+1. **Click "Code" button**
+2. Select preferred clone method and **copy Repo URL**
+3. **Open Git Bash** (or other terminal) and **enter "cd [Local Workstation Path]"**
+4. **Enter "git clone [Repo URL]"**
+5. **Enter "git checkout -b [New-Branch-Name]"**
+
+# Install Linters
+Linters exist to ensure a standard quality of code is maintained by all contributors. Run the following commands in your terminal to install/run the corresponding linter:
+
+## WEBHINT
+**install:** npm install --save-dev hint@7.x
+
+**run:** npx hint .
+
+## STYLELINT
+**install:** npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+**run:** npx stylelint "**/*.{css,scss}"
+
+## LIGHTHOUSE
+No install needed.
+## NODECHECKER
+**This ensures that "node_modules/" is included in .gitignore.** No install needed.
+
+
+# Author(s)
+👤 **Botlhale Setou**
+- GitHub: https://github.com/Botlhale-Setou
+- Twitter: https://twitter.com/Botlhale_Setou
+- LinkedIn: https://www.linkedin.com/in/botlhalesetou/
+
+## Show your support
+Give a ⭐️ if you like this project!
+
+## Acknowledgements
+- A special **thank you** to all those who helped shape the version we're on!
+- This design template is by [Cindy Shin](https://www.behance.net/adagio07)
+
+## 📝 License
```

Comment:
- [OPTIONAL] NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.